fe/fe-core/src/main/java/com/starrocks/mysql/privilege/Auth.java

```diff
@@ -1097,7 +1097,7 @@ private void grantImpersonateToRoleInternal(
     }
 
     // return true if user ident exist
-    private boolean doesUserExist(UserIdentity userIdent) {
+    public boolean doesUserExist(UserIdentity userIdent) {
```

Comment:
It's better to add a more concise message about the bug. The problem and solution should be described clearly.

Reply:
done